src/test.js

```diff
@@ -692,6 +696,49 @@ export function test( testName, callback ) {
 	newTest.queue();
 }
 
+function skip( testName, data ) {
+	if ( focused || config.currentModule.ignored ) {
+		return;
+	}
+
+	const test = new Test( {
+		testName: testName,
+		skip: true,
+		params: data
```

Comment:
Do we really need to pass `data` all the way through? (there's a few spots that maybe could be leaned down by removing them from calls and letting it cast to `undefined`s)

Reply:
Ah not for sure in this case. Removed here as well. Let me know if it looks better now.